src/main/java/io/jenkins/plugins/designlibrary/Links.java

```diff
@@ -28,7 +28,8 @@ public ContextMenu doContextMenu(StaplerRequest request, StaplerResponse respons
             // otherwise you can also programatically create them.
             // see the javadoc for various convenience methods to add items
             return new ContextMenu()
-                    .add(new MenuItem().withContextRelativeUrl("/").withIconClass("icon-up").withDisplayName("Back to Dashboard"))
+                    .add(new MenuItem().withContextRelativeUrl("/").withIconClass("symbol-arrow-up-outline plugins-ionicons-api")
```

Comment:
I suspect this isn't the right symbol, it looks like it's fallen back to a different one:
![image](https://github.com/jenkinsci/design-library-plugin/assets/21194782/a795e36f-5452-4336-b9f3-463c00120d51)

Reply:
Interesting behaviour, `Links#getIconFileName()` also returned the wrong definition for the plugin of the symbol, but still it is properly shown.